### PR TITLE
created step that creates a db

### PR DESCRIPTION
**Pull request by TheDogfather17**
Creates DD for uc feature. 
IAM to allow writing to publish bucket. 
DB step also writes data params passed by emr launcher to txt files to be picked up by another script

### CODE REVIEW

File: bootstrap_actions/cloudwatch.sh [hunk 1]
```diff
@@ -5,8 +5,8 @@ set -Eeuo pipefail
 cwa_metrics_collection_interval="$1"
 cwa_namespace="$2"
 cwa_log_group_name="$3"
-cwa_bootstrap_loggrp_name="$4"
-cwa_steps_loggrp_name="$5"
+cwa_bootstrap_loggrp_name="$5"
```
Comment (connoravo) [inline bootstrap_actions/cloudwatch.sh:8]: You've shifted the index but haven't made a variable for 4th arg input

File: steps/create_uc_feature_dbs.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    EXPORT_DATE="$8"
+
+    echo "$CORRELATION_ID" >>     /opt/emr/correlation_id.txt
+    echo "$S3_PREFIX" >>          /opt/emr/s3_prefix.txt
+    echo "$SNAPSHOT_TYPE" >>      /opt/emr/snapshot_type.txt
+    echo "$EXPORT_DATE" >>        /opt/emr/export_date.txt
+
+    log_wrapper_message "Creating UC Feature Databases"
+
+    hive -e "CREATE DATABASE IF NOT EXISTS ${uc_feature_db} LOCATION '${published_bucket}/${hive_metastore_location}';"
```
Comment (connoravo) [inline steps/create_uc_feature_dbs.sh:26]: What is creating the vars uc_feature_db, published_bucket and hive_metastore_location?

File: bootstrap_actions/cloudwatch.sh (hunk 1 above)
Comment (TheDogfather17) [reply bootstrap_actions/cloudwatch.sh:8]: there already is one, this was just ordered wrong

File: steps/create_uc_feature_dbs.sh (hunk 2 above)
Comment (TheDogfather17) [reply steps/create_uc_feature_dbs.sh:26]: its in the locals.tf

**Review by connoravo — APPROVED** [review]

**Pull request merged**